phpstan.neon.dist

```diff
@@ -2,6 +2,8 @@ includes:
     - phpstan-baseline.neon
 
 parameters:
+    parallel:
```

Comment:
I did this because I often get timeout errors. If it doesn't need to be here, I can delete it

Reply:
Let's check if the CI pass without it and if yes, remove it before merge